frontend/src/app/user/user-settings/user-settings.component.html

```diff
@@ -0,0 +1,65 @@
+<h2>User settings</h2>
+
+<p>Use the form below to change your name or password.</p>
+
+<form [formGroup]="form" (ngSubmit)="submit()">
```

Comment:
It's common that different settings connect to different models (in an applications where users have more to do) or that some things like changing your password/email have their own workflow for confirmation/verification.

So rather than using a single `<form>`, I usually to build a more modular settings menu.

For example, the user settings in LiNT are layed out like this:

![screenshot of settings menu of LiNT application](https://github.com/CentreForDigitalHumanities/lettercraft/assets/43678097/3a8ccab2-bf22-43c9-8f2e-dca30d1ba526)

My point is not about aesthetics, but the idea is to divide the page into several sections that can function as stand-alone components and use their own `<form>` (if applicable). For example, the change password option folds out into an *old password / new password / repeat new password* `<form>` that has its own submit event. This makes sense because (both here and in LiNT), changing your password connects to a different API endpoint from other settings.

Another advantage of dividing settings into several sub-components is that it's a component that can really grow out of control otherwise.

In this case, it would make sense to divide this page into the following sections:

- Username
- Personal details (i.e. first/last name)
- Email
- Password
- Delete account

Reply:
I really like this! This PR is getting big as it is, so I've created a separate issue for it: #67.